pkgs/by-name/ci/cinny-desktop/package.nix

```
@@ -67,12 +69,19 @@ rustPlatform.buildRustPackage rec {
         --set-key="Categories" --set-value="Network;InstantMessaging;" \
         $out/share/applications/cinny.desktop
     '';
+  postFixup = lib.optionalString stdenv.hostPlatform.isLinux ''
+    wrapProgram "$out/bin/cinny" \
+      --inherit-argv0 \
+      --set-default WEBKIT_DISABLE_DMABUF_RENDERER "1"
+      # --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ libayatana-appindicator ]}" \
+  '';
 
   nativeBuildInputs = [
     wrapGAppsHook3
     pkg-config
     cargo-tauri_1.hook
     desktop-file-utils
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```
This can also remove a bit of overhead from the script-based wrapper